english/travel-destinations/fall/fall.md

```diff
@@ -0,0 +1,5 @@
+Fall is the time to enjoy nature's amazing color displays known as "fall foliage"!
+
+Two great national parks for leaf peeping are 
+the [Blue Ridge Parkway](http://www.nps.gov/blri/index.htm) 
+and [Acadia](http://www.nps.gov/acad/index.htm)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.